### DISCUSSION TITLE
Causal structure learning algorithms

### DISCUSSION THREAD

**Discussion by kenneth-lee-ch** [agrumery/aGrUM#12] [Q&A] [answered]
Does PyAgrum implement the PC algorithm, conditional independence tests, and Meek Rules?

**Comment by phwuil** [comment, marked as answer]
I did not answer to this question :-) 
- conditionanl ind. tests are accessible via BNLearner [(see this notebook)](http://webia.lip6.fr/~phw/aGrUM/docs/last/notebooks/Chi2AndScoresFromBNLearner.ipynb.html)
- Meek rules are implemented but not in an accessible way (should change in not so long)
- PC algorithm is not implemented. However MIIC algorithm is a constraint-based algorithm that uses MutualInformation as a conditional independance test.
